src/compute-client/src/controller.rs

```diff
@@ -558,6 +558,7 @@ where
         }
 
         // Process pending replica metrics responses
+        // TODO(guswynn): remove the `replica_metrics`, field, its unused.
```

Comment:
This doesn't seem right?

Reply:
it isnt right! the compute client doesnt produce metrics anymore!

Reply:
Good catch! I opened a PR to remove it: https://github.com/MaterializeInc/materialize/pull/21025